### PR TITLE
blog post for lts20 and nightly ghc-9.4

### PR DESCRIPTION
**Pull request by juhp**
Please comment any tweaks or suggestions for improvement

### CODE REVIEW

**Comment by ysangkok** [comment]
It might be useful to mention that there is an easy workaround for people on Debian Bullseye : https://github.com/commercialhaskell/stack/issues/5881

**Review by mihaimaruseac — APPROVED** [review]

File: posts/announce-lts-20-nightly-ghc9.4.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+LTS 20 includes many [package changes](https://www.stackage.org/diff/lts-19.32/lts-20.0), and is the first LTS release with over 3000 packages!! Thank you for all the nightly contributions that made this possible.
+
+If your package is missing from LTS 20 and builds there, you can easily request to have it added using our straightforward process: just open a github issue in the [lts-haskell](https://github.com/commercialhaskell/lts-haskell/issues) project and following the steps in the template.
+
+## Stackage Nightly updated to ghc-9.4.3
+
+At the same time we are also excited to have moved Stackage [Nightly to GHC 9.4.3](https://www.stackage.org/nightly-2022-11-19) now!
+
+Almost 500 Nightly packages had to be [disabled](https://www.stackage.org/diff/nightly-2022-11-17/nightly-2022-11-19) as part of the upgrade to 9.4. Please help help to update your packages to build with ghc-9.4 and get them back into Stackage Nightly, thank you!
```
Comment (psibi) [inline posts/announce-lts-20-nightly-ghc9.4.md:20]:
```suggestion
Almost 500 Nightly packages had to be [disabled](https://www.stackage.org/diff/nightly-2022-11-17/nightly-2022-11-19) as part of the upgrade to 9.4. Please help us to update your packages to build with ghc-9.4 and get them back into Stackage Nightly, thank you!
```

**Review by psibi — APPROVED** [review]

File: posts/announce-lts-20-nightly-ghc9.4.md (hunk 1 above)
Comment (juhp) [reply posts/announce-lts-20-nightly-ghc9.4.md]: Thanks!

**Comment by juhp** [comment]
> It might be useful to mention that there is an easy workaround for people on Debian Bullseye

@ysangkok good point - I added some words about the glibc-2.32 issue. Hope that helps users a bit.

Since it is a bit complicated and it may take time for stack to solve this - it might actually warrant a separate detailed blogpost.  Right now the simplest solution seems to me to use ghcup's stack ghc integration script/feature?

**Review by snoyberg — APPROVED** [review]

**Pull request merged**